package.json

```diff
@@ -64,11 +64,11 @@
     "mocha": "^7.0.0"
   },
   "dependencies": {
-    "@hapi/crumb": "^7.2.3",
-    "@hapi/hoek": "^8.0.2",
-    "@hapi/joi": "^16.1.7",
+    "@hapi/crumb": "^8.0.0",
+    "@hapi/hoek": "^9.0.4",
     "@vrbo/steerage": "^8.4.0",
```

Comment:
Probably ought to update [steerage](https://github.com/expediagroup/steerage) first.